partition/roles/mgmt-server/templates/ssh_config.j2

```diff
@@ -0,0 +1,8 @@
+{% for host in groups.all %}
```

Comment:
This loop can be very exhaustive. Instead provide a `mgmt_server_ssh_config_groups: []` variable and let the user decide which hosts to add.